bin/scrape_software_versions.py

```diff
@@ -6,14 +6,10 @@
 regexes = {
     'nf-core/mhcquant': ['v_pipeline.txt', r"(\S+)"],
     'Nextflow': ['v_nextflow.txt', r"(\S+)"],
```

Comment:
You still havent added in version commands for all of the software in the environment.
See: https://github.com/nf-core/atacseq/blob/master/bin/scrape_software_versions.py

Reply:
done